core/tests/webdriverio_utils/ExplorationEditorMainTab.js

```diff
@@ -887,7 +886,9 @@ var ExplorationEditorMainTab = function() {
 
     // Wait for state name container to completely disappear
     // and re-appear again.
-    await waitFor.visibilityOf(
+    // eslint-disable-next-line oppia/e2e-practices
+    await browser.pause(2000);
+    await waitFor.presenceOf(
```

Comment:
Why are you using presence of keeping in mind that earlier visibility was checked?

Reply:
Reverted.